src/amber/cli/templates/app/config/environments/development.yml.ecr

```diff
@@ -19,7 +19,7 @@ redis_url: "redis://localhost:6379"
 when "mysql" -%>
 database_url: mysql://root@localhost:3306/<%= database_name %>_development
 <% when "pg" -%>
-database_url: postgres://<%= `whoami`.strip("\n") -%>:@localhost:5432/<%= database_name %>_development
```

Comment:
This is the `whoami` I mentioned in another that I added to make getting started easier. I'd prefer to leave this in, unless we absolutely need it for docker to work. And even then I'd like to open another issue so we can keep the convenience of the `whoami` working for those on native installs and not docker.

Reply:
Understood. Whoami using the logged in user of your local system.  This is not necessarily the user setup for postgres.  I log into a separate box for development.  Let's see if we can come up with another solution.

Reply:
Let's just bring this in as-is for now and make the `whoami`/default db user stuff another issue